docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/jira.md

```diff
@@ -0,0 +1,659 @@
+
+#### Output
+
+| Name  | Type     | Description                                           |
+| ----- | -------- | ----------------------------------------------------- |
+| `key` | `string` | The key/id of the created Jira issue (e.g., EXT-123). |
+
+#### Example
+
+This example creates a new Jira issue to track a promotion, assigns it to a team member, and adds relevant labels.
+
+```yaml
+steps:
+- uses: jira
+  as: create-promotion-issue
+  config:
+    credentials:
+      secretName: jira-credentials
+    createIssue:
+      projectKey: DEPLOY
+      summary: "Deploy ${{ imageFrom(vars.imageRepo).Tag }} to ${{ ctx.stage }}"
+      description: "Deploying ${{ imageFrom(vars.imageRepo).RepoURL }}:${{ imageFrom(vars.imageRepo).Tag }} to ${{ ctx.stage }} environment. Promotion ID: ${{ ctx.promotion }}. Freight: ${{ ctx.targetFreight.name }}."
+      issueType: Task
+      assigneeEmail: devops@company.com
+      labels:
+      - promotion
+      - "${{ ctx.stage }}"
+      - "release-${{ imageFrom(vars.imageRepo).Tag }}"
+# Use the created issue key in subsequent steps
+- uses: jira
+  config:
+    credentials:
+      secretName: jira-credentials
+    updateIssue:
+      issueKey: "${{ outputs['create-promotion-issue'].key }}"
+      status: "IN PROGRESS"
+```
+
+### Update Issue
+
+Updates an existing Jira issue with new information.
+
+#### Configuration
+
+| Name                         | Type     | Required | Description                                                       |
+| ---------------------------- | -------- | -------- | ----------------------------------------------------------------- |
+| `updateIssue.issueKey`       | `string` | Y        | The Jira Issue Key (e.g., EXT-123).                               |
+| `updateIssue.summary`        | `string` | N        | Updated summary or title of the issue.                            |
+| `updateIssue.description`    | `string` | N        | Updated description. Supports markdown formatting.                |
+| `updateIssue.adfDescription` | `object` | N        | ADF content for complex formatting. Alternative to `description`. |
+| `updateIssue.issueType`      | `string` | N        | Updated issue type.                                               |
+| `updateIssue.assigneeEmail`  | `string` | N        | Email of the user to assign the issue to.                         |
+| `updateIssue.status`         | `string` | N        | Status to set for the issue (e.g., 'IN PROGRESS', 'DONE').        |
+| `updateIssue.addLabels`      | `array`  | N        | Labels to add to the issue.                                       |
+| `updateIssue.removeLabels`   | `array`  | N        | Labels to remove from the issue.                                  |
+| `updateIssue.customFields`   | `object` | N        | Custom fields to update.                                          |
+
+#### Output
+
+This step does not produce any output values.
```

Comment:
"Output values" sounds redundant, although it's not a hill I would die on. Take it or leave it. Applies in other places as well.

```suggestion
This step does not produce any output.
```